### PR TITLE
Add Travis CI config

### PR DESCRIPTION
**Pull request by webknjaz**
Supersedes #2

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    python: "3.7"
+    before_install:
+    - choco install python --version 3.7
+    env:
+      PATH: >-
+        /c/Python37:/c/Python37/Scripts:$PATH
+
+  - name: Publish dists to production PyPI
+    stage: Publish dists to PYPI
+    if: tag IS present
```
Comment (webknjaz) [inline .travis.yml]: @Mariatta with this any tag will work, no need to construct is as `cherry-picker-smth..`, just tag with the version number as in `v1.0.0` or just `1.0.0` but better decide on a single consistent style now.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      # built by during `install` step above.
+      skip-cleanup: true
+      # `skip-existing: true` is required to skip uploading dists, already
+      # present in PyPI instead of failing the whole process.
+      # This happenes when other CI (AppVeyor etc.) has already uploaded
+      # the very same dist (usually sdist).
+      skip-existing: true
+      user: &pypi-user core-workflow
+      password: &pypi-password
+        # Encrypt with `travis encrypt -r python/core-workflow --org` while using travis-ci.org;
```
Comment (Mariatta) [inline .travis.yml]:
```suggestion
        # Encrypt with `travis encrypt -r python/cherry-picker --org` while using travis-ci.org;
```

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        # change to `travis encrypt -r python/core-workflow --api-endpoint 'https://api.travis-ci.com/'`
```
Comment (Mariatta) [inline .travis.yml]:
```suggestion
        # change to `travis encrypt -r python/cherry-picker --api-endpoint 'https://api.travis-ci.com/'`
```

File: .travis.yml (hunk 2 above)
Comment (Mariatta) [reply .travis.yml]: Actually this part is not needed. We're using travis-ci.com

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        # upon switch to __free__ travis-ci.com:
+        secure: >-
+          ec2BNgqhpXEY2mdNCmfs+lfufIpMAyvOMLDPyikJBwTkmEAQWEmBxNPt67vnun6ODKVuneYYvZCxnpcvASii/2DoBomVhWCVqmFQk888ehG31gKBhcRfTYePZ9B/1GNLchj+Z53eTJX3LVZBur6eICK1DxmgjHKLFSqYKCdxmuw+S8ethDzDE6cX5ce6D2+KDBMN2tUc1C9t15Q9sk8EgElY1JwRaiJ28uZsNfZQcP3rkusbDYPK5mFFJIv6xgEQbA31EvHU9/zvHYf7KwtkkgQzV4wN+FCr/ekGp9T8YneFSrioCY6nGznzsHcKXJsUuJEKBvjwOxy26/RfO/O+0cN6b3fGmI9wQVAnhbd/WszdN0/RmOtwXcJS/uxDTZSPcrXTKRqWaESKt9tGDXeFKLyrpi5Rc9wnxonVzwAOKZSody+ADG6EfLNsVJZt+RM3nV9aOgS20pCctsgqgW+qYBkmpYuAbIyaZRe3oi6lzYWXfZcwN2JIgiy7LVNn+JBPmNT3W21oR13TA7GDy/sWec9Odds4uZXUqfpkdVl+9/TzugH1VtgzhOyC/nU7y/m0Uo5q4QYTkAEZUrYP1N/FEUnvwLRHsqUzWk5beJtycOsHoOj7O9KG1g9gWfBxjfN3N5D2n1P7Xvf4dwcKc+mBVmq6tjAQYgqSdYttKrQPxrw=
```
Comment (Mariatta) [inline .travis.yml]:
```suggestion
          lBje6oa0VUCeG/btHJbNMc5LzH8BhjhmpP02S6PiCpXP6ZfhpP8cjaK+kEXkUWcdxmztnJYogVTE4efS9xSGO9bgJ37h8KgbOwRueUJpypZwjQiYECnJWHDqEQZlu8JXxHTrm3caAWMAFp4Volduzum/UiHAHLaaqzl9y2Sgzqkx1bm4u0r8Cb79fmUR5MxIFEXcK/TblVHvXXimWSGn5zWlRoin1xO6a0lTkYFt3XAE/uPWDwDWUl5KmLGoNd7HKJqx6TfOAbJIcu1kNBxqMYgu849Q2MwTherkKrTtXtKVuZaSX2ZrydO7dAWQpFvcb0Dd/VWAwQpbHrQJPfh0Uzxpv/t0KoNNWvrjRx2KnqUcphhOWBiv18aLaYBBUrMahyUKk1SifM/W/JdJ/RjMRNBlmzkqgpviNgIDJU26cSORFC2xQzgb+cPA8HUsqvZaJiCAQ/Zz4PZYib+3cM+Vfp1q/hAfREFnuojWEDXddPuiXqgZissHrsybQ+K4sHymioV8MW/nR1k6e3lUDjIkeuwxFANA3JIdunTWmy4Pm3ItPi2DV1hEIcLGFjO5Oa4P2QbHcQBnCgK7XdInFAjF0sf6ZmUvJcKFiVm7UODBj5zlYHaKKdOJTe3kLFP5iwinIk5CZCps34WYPvvXvFvLSamv7caxh4ySVupPn85voYQ=
```

File: .travis.yml (hunk 1 above)
Comment (Mariatta) [reply .travis.yml]: Let's go with `vX.Y.Z`.

File: .travis.yml (hunk 2 above)
Comment (webknjaz) [reply .travis.yml]: Well, it's good for config readers in the future. For educational purposes, for example :)

File: .travis.yml (hunk 1 above)
Comment (webknjaz) [reply .travis.yml]: ok. but travis ci config doesn't need to validate that.

**Comment by Mariatta** [comment]
Thanks so much!

**Comment by webknjaz** [comment]
@Mariatta you're welcome :)

FTR when you'll try releasing a new version just be aware that sometimes that encrypted password secret could be generated wrongly (this happened to me a lot of times and the reason is not completely clear) so it may need updating in such case.

**Pull request merged**